developer/code-reviews.md

```diff
@@ -0,0 +1,22 @@
+# Code Review
+
+Code contributions are reviewed by the Reaction core and community contributors.
+All merge requests are reviewed and approved by at least one core contributor before merge.
+
+Here are is a brief checklist of common items reviewed in every merge.
+
+-   Code style guide adhered to and **linted** (`eslint`, 0 errors, no ignores)
```

Comment:
There are some API's where using an eslint-disable-line is necessary because the api requires underscored rather than camelcase arguments